sql/ddl_api.sql

```diff
@@ -308,3 +308,155 @@ BEGIN
     PERFORM _timescaledb_internal.attach_tablespace(hypertable_id, tablespace);
 END
 $BODY$;
+
+
+CREATE OR REPLACE FUNCTION reindex_indexes(
+    index_oids OID[],
+    verbose   BOOLEAN = FALSE,
+    recreate  BOOLEAN = FALSE
+)
+    RETURNS VOID LANGUAGE PLPGSQL VOLATILE AS
+$BODY$
+DECLARE
+    new_index_oids OID[];
```

Comment:
are we aligning types any more? I've seen some inconsistency recently and was wondering if we are no longer doing that

Reply:
what do you mean?